healthcareai/common/transformers.py

```diff
@@ -54,7 +56,17 @@ def transform(self, X, y=None):
         # Return if not imputing
         if self.impute is False:
             return X
-
+        
+        #Warn users if %nan is too high
+        for c in X:
+            pct_impute = X[c].isnull().sum() / len(X)
+            if pct_impute > self.max_impute:
+                warnings.warn("{0:.2f}% of data for column '{1}' is missing. Imputed "
```

Comment:
This might read a little better if you state the column name first, then the %missing. I totally love this, and I'm going to steal your idea of using warnings for a few other thigns!